packages/ipfs-core/test/name.spec.js

```diff
@@ -50,6 +50,26 @@ describe('name', function () {
       expect(republisher._republishEntries.calledTwice).to.equal(true)
     })
 
+    it('should not republish self key twice', async function () {
+      const mockKeychain = {
+        listKeys: () => Promise.resolve([{ name: 'self' }])
+      }
+      republisher = new IpnsRepublisher(sinon.stub(), sinon.stub(), sinon.stub(), mockKeychain, {
+        initialBroadcastInterval: 500,
+        broadcastInterval: 1000,
+        pass: 'pass'
+      })
+      republisher._republishEntry = sinon.stub()
+
+      await republisher.start()
```

Comment:
Does this need stopping?

Reply:
It is stopped in the afterEach (line 29), like remaining tests